crux-bench/src/crux/bench/watdiv_crux.clj

```diff
@@ -23,22 +23,46 @@
          (map (juxt :query-idx identity))
          (into {}))))
 
-(defn ->query-result [watdiv-query-results]
-  (let [both-completed (->> watdiv-query-results (filter (every-pred :rdf4j-result-count :result-count)))
-        crux-correct (->> both-completed (filter #(= (:rdf4j-result-count %) (:result-count %))))
+(def db-query-results
+  {"rdf4j" (some->
+            (bench/load-from-s3 "rdf4j-3.0.0/rdf4j-20200214-174740Z.edn") parse-results)
+   "neo4j" (some-> (bench/load-from-s3 "neo4j-4.0.0/neo4j-20200219-114016Z.edn") parse-results)})
+
+(defn- db-result-count [db-name]
```

Comment:
don't reckon you need to construct these keys - it'd be unambiguous to have (e.g.) `{:rdf4j {:crux-time-taken-ms ..., :db-time-taken-ms ...}}` because it's under the `:rdf4j` key